src/features/unified-share-modal/UnifiedShareForm.js

```diff
@@ -482,7 +482,7 @@ class UnifiedShareForm extends React.Component<USFProps, State> {
             className: 'usm-ftux-tooltip',
             // don't want ftux tooltip to show if the recommended sharing tooltip callout is showing
             isShown: !recommendedSharingTooltipCalloutName && shouldRenderFTUXTooltip && showCalloutForUser,
-            position: 'middle-left',
+            position: 'bottom-center',
```

Comment:
it's possible that with a long tooltip text, the problem is now moved to the bottom of the modal i.e. the tooltip becomes off-screen at the bottom.

might not be reproducible with english but could be with other languages.

Reply:
but we don't really have a better solution at the moment besides removing it altogether